### ISSUE TITLE
ROI does not account for periodic withdrawals to your eth1 address (Capella Upgrade)

### ISSUE THREAD

**Issue by augchan42** [raskitoma/pulse-staking-dashboard#7]
I notice the dashboard doesn't calculate ROI properly, as it believes any withdrawal is due to slashing or something similar.   Is there a way to configure the dashboard to ignore the periodic sweeps that Capella Upgrade supports?

From: https://lighthouse-book.sigmaprime.io/partial-withdrawal.html

"if a validator has a withdrawal credential type 0x01, any rewards above 32ETH will be periodically withdrawn to the withdrawal address. This is also known as the "validator sweep", i.e., once the "validator sweep" reaches your validator's index, your rewards will be withdrawn to the withdrawal address. At the time of writing, with 560,000+ validators on the Ethereum mainnet, you shall expect to receive the rewards approximately every 5 days."

**Comment by raskitoma** [comment]
Hi, I'm trying to figuring out what are the correct data sources. Everything I got it's from the node metrics via prometheus, but I cannot find a proper variable to get data this way. If you have any idea, please share it with me to improve this dashboard.

**Comment by augchan42** [comment]
From my check the prometheus metrics give a snapshot of the current state (what is my validator balance now).  Any increase or decrease is inferred by the dashboard from that snapshot.  Maybe you can configure the backend aggregator in the dashboard to ignore all withdrawals?  But then you'd need to account for slashing, and other corner cases, etc

**Comment by raskitoma** [comment]
@augchan42 you are right. I just was looking everywhere for a solution for this... and the answer is the same: I need to configure the aggregator.

Problem is, I will miss a lot of data if I do that, because I think I need to redo all again (prometheus, etc.)

That's maybe because I have no experience with prometheus... If you can, guide me on how to do this, please!
